### ISSUE TITLE
Question: Multiple simultaneous Bluetooth connections to different devices?

### ISSUE THREAD

**Issue by PawelGIX** [randdusing/cordova-plugin-bluetoothle#767]
1. Does this plugin support multiple simultaneous Bluetooth connections to different devices?
2. It is it still actively developed and does it work properly on iOS Android?

**Comment by asif166037** [comment]
@PawelGIX  Yes it possible. to connect the multiple device on one single central mode
